python/mdvtools/dbutils/mdv_server_app.py

```
@@ -557,60 +703,247 @@
+                        user_info = auth0_provider.get_user(token)
+                        if user_info:
+                            return jsonify(user_info)
+                        else:
+                            return jsonify({"error": "Failed to retrieve user information from Auth0."}), 500
+                    else:
+                        return jsonify({"error": "Not authenticated with Auth0."}), 401
+
+                elif auth_method == 'shibboleth':
+                    # Retrieve user attributes from available Shibboleth headers
+                    eppn = request.headers.get('Shibboleth-Eppn')  # e.g., whgu1064@ox.ac.uk
+                    x_forwarded_user = request.headers.get('X-Forwarded-User')  # e.g., whgu1064@ox.ac.uk
```

Ruff (0.8.2)

Local variable `x_forwarded_user` is assigned to but never used

Remove assignment to unused variable `x_forwarded_user`

(F841)